### PR TITLE
introduce var_meter

### LINKED ISSUE #112: Create symbol for var_meter

### PR DESCRIPTION
**Pull request by bhavyajain511**
fixes #112

### CODE REVIEW

**Comment by bhavyajain511** [comment]
fixes #113

File: symbols/var_meter_horz.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { defineSymbol } from "drawing/defineSymbol"
+import svgJson from "assets/generated/var_meter.json"
+
+const { paths, texts, bounds, refblocks, circles } = svgJson
+
+export default defineSymbol({
+  primitives: [
+    ...Object.values(paths),
+    ...Object.values(circles),
+    {
```
Comment (imrishabh18) [inline symbols/var_meter_horz.ts:10]: @bhavyajain511 I don't think this `{REF}` and `{VAL}` is needed, cause this is already there in the json which is getting imported

**Comment by imrishabh18** [comment]
Looks good! just added one comment. I think it will good to merge after that

File: symbols/var_meter_horz.ts (hunk 1 above)
Comment (imrishabh18) [reply symbols/var_meter_horz.ts:10]: Refer the `capacitor_horz` file

**Review by imrishabh18 — CHANGES_REQUESTED** [review]

**Pull request merged**